crates/vm/src/system/connector/mod.rs

```diff
@@ -55,6 +57,10 @@ impl<F: Field> BaseAirWithPublicValues<F> for VmConnectorAir {
     fn num_public_values(&self) -> usize {
         VmConnectorPvs::<F>::width()
     }
+
+    fn columns(&self) -> Vec<String> {
```

Comment:
VmConnectorPvs is ok? Why not ConnectorCols?

Reply:
Yes, it should be the columns, not the public values, I think! Basically, it always depends on what `Air::eval` does: It always gets `let cols: MyCols = builder.main().row_slice(...).borrow()`. Then `MyCols` should be the struct providing the names, which in this case is `ConnectorCols`.